### ISSUE TITLE
use alternate hosting service for the models?

### ISSUE THREAD

**Issue by April93** [haoheliu/voicefixer#43]
Can't seem to download them. could you just please put the files on huggingface? These weird chinese download sites either are painfully slow or don't seem to work.

**Comment by haoheliu** [comment]
Does this link work for you? https://zenodo.org/record/5600188#.ZA3UpezP1sU

**Comment by April93** [comment]
No. download is way too slow. Why not just a proper filehost? huggingface, mega, mediafire, etc?

**Comment by haoheliu** [comment]
If you found downloading on zenodo slow (which is rare), you can try to download it on google colab and then download it to your computer. I'll make upload it to HF when I got time.

**Comment by April93** [comment]
colab seems to save the files to /root/.cache which I'm not given access to so I can't download to my local machine.

**Comment by haoheliu** [comment]
I mean in the colab, you can use the following command to download model (colab might have better network connection with zenodo):
```shell
! wget https://zenodo.org/record/5600188/files/epoch%3D15_trimed.ckpt?download=1
```
